### ISSUE TITLE
[Feature Request] Make getting new access_token easier from POSTMAN

### ISSUE THREAD

**Issue by tledwar** [awslabs/fhir-works-on-aws-deployment#190]
I am having issues using Postman to make my first call to FHIR Works on AWS. Per https://github.com/awslabs/fhir-works-on-aws-deployment#retrieving-access-token-via-postman-scope--openid-profile and specifically in the section called Authorizing a User, I am use the Get New Token feature. Please note I am using the Postman collection from this project.

For this feature, Postman opens up a dialog box which I have done my best to fill out.

There is no mention of a Grant Type, so I have assumed Password Credentials since we are trying to access this by username and password.

My first issue is if Token Name is supposed to specially be called anything.

My username and password is from the setup in cognito when installed using cloud formation for FHIR Works on AWS.

Bottom line is that I have not been able to gain authorization using the directions provided. The URL endpoints all look correct and came from other documentation such as the implementation guide.

**Comment by Bingjiling** [comment]
Hi @tledwar , thanks for reaching out! Could you please share a screenshot of the Postman dialog box or any input fields you're having trouble with?

**Comment by tledwar** [comment]
Attached  is the screen shot for the Get New Token.

![Untitled](https://user-images.githubusercontent.com/15660075/105380585-acc8cd00-5bd3-11eb-98da-dc0212a34209.png)

When you request a new token, you get the following error:
Could not complete OAuth 2.0 login. Check Postman Console for more details.

Since Cognito is Oath2, I was thinking Oath 2 should be used as the Authorization. If I get a token through aws cognito-idp initiate-auth --auth-flow USER_PASSWORD_AUTH --client-id ... and use  Bearer Token, the API call works fine. Its just much more convenient to get a new token through the Postman interface, but maybe it is not.

**Comment by rsmayda** [comment]
> Its just much more convenient to get a new token through the Postman interface, ...

I think that is right and the right piece of feedback. Thanks for this, we will take a look into these Quality of life improvements

**Comment by KevinMayfield** [comment]
I'm working with a java client. I'd ideally like to use client_credentials grant but believe this doesn't work with the default RBAC. 

Should this password credentials grant work with Cognito? It doesn't appear to. 

In postman I get an `invalid client` error.

**Comment by nisankep** [comment]
FHIR Works on AWS has been moved to maintenance mode. While in maintenance, we will not add any new features to this solution. All security issues should be reported directly to AWS Security at [aws-security@amazon.com] (mailto:security@amazon.com). If you are new to this solution, we advise you to explore using [HealthLake] (https://aws.amazon.com/healthlake), which is our managed service for building FHIR based transactional and analytics applications. You can get started by contacting your AWS Account team. If you are an existing customer
of FHIR Works on AWS, and have additional questions or need immediate help, please reach out to [fwoa-migration-support@amazon.com](mailto:fwoa-migration-support@amazon.com) or contact your AWS Account team.
